src/span-data.js

```diff
@@ -92,7 +97,10 @@ SpanData.prototype.createChildSpanData = function(name, skipFrames) {
 };
 
 SpanData.prototype.addLabel = function(key, value) {
-  this.span.setLabel(key, value);
+  key = traceUtil.truncate(key, constants.TRACE_SERVICE_LABEL_KEY_LIMIT);
+  var val = typeof value === 'object' ? util.inspect(value) : '' + value;
```

Comment:
If `value` is a Symbol, string concatenation will throw. Can you add tests for all non-object types?